pyTigerGraph/gds/dataloaders.py

```diff
@@ -502,6 +502,7 @@ def _set_kafka_topic(self) -> None:
 
     @staticmethod
     def _request_kafka(
+        self,
```

Comment:
Why is self needed here?

Reply:
Because I thought it was needed to not be a static function at some point... removed.